core/src/test/java/org/springframework/security/provisioning/JdbcUserPasswordDetailsManagerTests.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2002-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.security.provisioning;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests for {@link JdbcUserPasswordDetailsManager}
+ *
+ * @author Geir Hedemark
+ */
+public class JdbcUserPasswordDetailsManagerTests extends JdbcUserDetailsManagerTests {
```

Comment:
Like `JdbcUserPasswordDetailsManager`, this should not use deprecated classes.

Reply:
Done